test.py

```diff
@@ -132,6 +151,7 @@ def main():
     token_id = create_token(client, operator_id)
     associate_token(client, recipient_id, recipient_private_key, token_id)
     transfer_token(client, recipient_id, token_id)
+    delete_token(client, token_id)
```

Comment:
`delete_token(client, token_id, admin_key)`

-> can just be a new key generated within the script (for an example see: new_account_private_key)